src/components/EditPasswordForm/EditPasswordForm.tsx

```diff
@@ -27,13 +29,32 @@ type Props = {
 };
 
 const EditPasswordForm: React.FC<Props> = ({ onSubmit, onChange, onBlur, showOldPasswordField, showResetTokenField, value, errors, submitting }: Props) => {
-  const { t } = useTranslation('account');
+  const { t } = useTranslation(['account', 'user']);
+  const email = useAccountStore((state) => state.user?.email);
+  const [errorMessage, setErrorMessage] = useState<string>();
+  const feedbackMessage = errors.form || errorMessage;
+  const resendEmailClickHandler = async () => {
+    try {
+      if (email) {
+        await resetPassword(email, '');
+      }
+    } catch (error: unknown) {
+      setErrorMessage(t('user:account.resend_mail_error'));
+    }
+  };
```

Comment:
This logic needs to be placed up one level (container). This change adds store logic to a component.

Reply:
You can then also combine the `errorMessage` with the form `errors` nicely.